bin/http-server

```diff
@@ -7,6 +7,7 @@ var colors     = require('colors'),
     httpServer = require('../lib/http-server'),
     portfinder = require('portfinder'),
     opener     = require('opener'),
+    columnify  = require('columnify'),
```

Comment:
I would prefer that we either don't add a new dependency, or use it for all logging (and the help output).

Reply:
I am not a big fan of adding this dependency. It does make the headers look better, but I'm not sure we need the extra baggage to do that. I think `util.format` should do just fine without any extra libraries